lib/shell/shellscript.js

```diff
@@ -74,6 +74,18 @@ class ShellScriptWorker {
             } else {
                 console.log('No content type information file found');
             }
+
+            if (await fs.pathExists(preparedFiles.optionsFile)) {
+                try {
+                    const options = await fs.readJSON(preparedFiles.optionsFile);
+                    if (options.postProcess === true || options.postProcess === "true") {
+                        rendition.postProcess = true;
```

Comment:
The original idea was to make this general and set the [options of the worker](https://git.corp.adobe.com/nui/asset-compute-sdk/blob/master/lib/worker.js#L62).

The downside is that since it only happens after the callback ran, we have to modify it after the fact and it cannot impact options that are needed before (say `disableSourceDownload`).

And since `rendition.postProcess` is not even a normal `option` it doesn't really work out...

So yeah, maybe this is ok for now.